2.7.pylintrc

```diff
@@ -8,28 +8,38 @@
 # attribute-defined-outside-init<W0201>: (hi-pri) Used when an instance attribute is defined outside the __init__ method.
 # bad-continuation<C0330>: (needs review) **REMOVED in pylint-2.6.0**
 # bad-indentation<W0311>: (hi-pri) Used when an unexpected number of indentation's tabulations or spaces has been found.
+# bad-open-mode<W1501>: (hi-pri) Python supports: r, w, a[, x] modes with b, +, and U (only with r) options.
```

Comment:
this is a comment for line 9 above

># bad-continuation<C0330>: (needs review) **REMOVED in pylint-2.6.0**

maybe we should disable this one globally (to the entire project)?

Reply:
We can definitely do that.

Note that python2.6 and python3.4 are stuck on earlier versions that _do_ have this symbol (any of 3.[6,7,8] might also get stuck on current stable, 2.5.3, which also has this symbol), so I'm not sure if that changes things. We could potentially leave this enabled in this file, and disable it for other versions once they update.